charts/jaeger/Chart.yaml

```
@@ -4,6 +4,8 @@ description: A Jaeger Helm chart for Kubernetes
 name: jaeger
 type: application
 version: 4.0.0
+annotations:
+  Jaegerv1Version: "1.53.0"
```

```suggestion
  Jaegerv1Version: "1.62.0"
```
use latest where agent was present